Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -28,6 +43,10 @@ open class ChatChannelListVC<ExtraData: UIExtraDataTypes>: UIViewController,
     override open func viewDidLoad() {
         super.viewDidLoad()
         
+        title = "Stream Chat"
+        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: userAvatarView)
+        navigationItem.rightBarButtonItem = UIBarButtonItem(customView: createNewChannelButton)
+        
```

Comment:
Should we extend `ChatChannelListVC` with `AppearanceSetting` and do this there?

Reply:
done in d4cc33bf143609304b77a3d9b43e8c6ad2086893